test/src/eks_blueprints_e2e_test.go

```diff
@@ -120,7 +120,7 @@ func TestEksBlueprintsE2E(t *testing.T) {
 			inputTfOptions := &terraform.Options{
 				/*The path to where our Terraform code is located*/
 				TerraformDir: tempExampleFolder,
-				VarFiles:     []string{testCase.name + ".tfvars"}, // The var file paths to pass to Terraform commands using -var-file option.
+				// VarFiles:     []string{testCase.name + ".tfvars"}, // The var file paths to pass to Terraform commands using -var-file option.
```

Comment:
if we are not using the `tfvars` then we need align the cluster name through variables. Currently tfvars using the cluster name as `aws-terra-test-eks` and we are using this to validate the terra tests.

Updating the `variables.tf` should fix the issue `tenant=aws`, `environment=terra` and `zone=test`

Reply:
thanks for the catch - I updated terratest to set the variables to match the expected cluster ID as you've shown above. I figure this is probably safer in case both terratest and the e2e-parallel are deployed at the same time (avoid name conflicts). Fixed in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/544/commits/e32e19a332a0d984e7c696607736624b8109f3ad - let me know what you think

Reply:
And https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/544/commits/289acf5080d9fd95a22a899da80c42924485c349 - forgot it was in two places

Reply:
Thanks @bryantbiggs! This looks great 👍🏼